loops/csharp/code.cs

```diff
@@ -0,0 +1,16 @@
+﻿var u = int.Parse(args[0]);
+var r = Random.Shared.Next(10_000);
+
+//Use span instead of array
+Span<int> a = new int[10_000];
```

Comment:
It might have little effect though (allocation occurs just one time)

https://github.com/bddicken/languages/pull/21#discussion_r1859017145

Reply:
Yup - it's a pointer bump and zeroing in either case (of course goes through an allocation call but it's practically noise). Also there's a trick to avoid the cast or target typing for stackalloc - if you place it in parentheses it will give you a span: `var buf = (stackalloc byte[32]);`.

Reply:
FWIW, I was about to suggest stackalloc too, but I benchmarked it and I confirmed it doesn't make a difference in this case. 

Haven't tried the parenthesis, but I think you get the exact same thing with explicit typing Span<int> instead of var.

Reply:
There would be no difference, I just didn't know about parentheses until later too and it looks visually annoying to have that one single span to not use `var` among other variables 😅

The execution time here is completely dominated by the loop itself and 1-2s is an eternity compared to ~40KiB array allocation.